src/main/java/core/selenium/webdrivers/Chrome.java

```diff
@@ -0,0 +1,41 @@
+package core.selenium.webdrivers;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+/**
+ * Chrome class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public final class Chrome implements Browser {
+
+    private static Chrome chrome;
+    private WebDriver driver;
+
+    /**
+     * Constructor of Chrome class.
+     */
+    private Chrome() {
+        System.setProperty("webdriver.chrome.driver", "src/main/java/core/selenium/webdrivers/driver/chromedriver");
```

Comment:
Maybe you can use in this path enum or variables .

Reply:
Thank you, this wrong was fixed.